benchmark/storage/parallel_scan_benchmark.cpp

```diff
@@ -0,0 +1,193 @@
+#include <functional>
+#include <limits>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+#include "benchmark/benchmark.h"
+#include "benchmark_util/benchmark_config.h"
+#include "catalog/catalog.h"
+#include "catalog/catalog_defs.h"
+#include "common/scoped_timer.h"
+#include "execution/exec/execution_context.h"
+#include "execution/sql/table_vector_iterator.h"
+#include "execution/table_generator/table_generator.h"
+#include "loggers/loggers_util.h"
+#include "main/db_main.h"
+#include "storage/storage_util.h"
+#include "test_util/storage_test_util.h"
+#include "transaction/deferred_action_manager.h"
+#include "transaction/timestamp_manager.h"
+#include "transaction/transaction_context.h"
+
+#include "execution/compiler/compiler.h"
+#include "execution/compiler/expression_util.h"
+#include "execution/compiler/output_checker.h"
+#include "execution/compiler/output_schema_util.h"
+#include "execution/exec/output.h"
+#include "execution/executable_query.h"
+#include "execution/execution_util.h"
+#include "execution/sema/sema.h"
+#include "execution/sql/value.h"
+#include "execution/vm/llvm_engine.h"
+#include "execution/vm/module.h"
+#include "planner/plannodes/seq_scan_plan_node.h"
+#include "type/transient_value.h"
+#include "type/transient_value_factory.h"
+#include "type/type_id.h"
+
+namespace terrier {
+class ParalleScanBenchmark : public benchmark::Fixture {};
```

Comment:
It might be helpful to include a setup function that sets a bunch of class variables due to the repetition of creating the benchmark variables (db_main, catalog, txn_manager, etc.)

Reply:
We encountered a weird bug when taking the setup code out in a separate function so we jut put them in here. We'll give it another shot and will update it in the next commit.